docs/src/substructure.md

```diff
@@ -0,0 +1,129 @@
+# Jet Substructure
+
+## Structures
+
+### `MassDropTagger`
+
+The `MassDropTagger` structure is used for tagging jets based on mass drop conditions, which helps in identifying subjets within a jet that undergo a significant drop in mass.
+
+**Fields**:
+
+- `mu::Float64`: Maximum allowed mass ratio for a jet to pass the tagging condition.
+- `y::Float64`: Minimum kT distance threshold for parent jet separation.
+
+---
+
+### `SoftDropTagger`
+
+The `SoftDropTagger` instance is used to apply soft-drop grooming to jets, removing soft, wide-angle radiation. This approach is commonly used in jet grooming to reduce contamination from soft particles.
+
+**Fields**:
+
+- `zcut::Float64`: Minimum allowed energy fraction for subjets.
+- `b::Float64`: Angular exponent controlling soft radiation suppression.
+- `cluster_rad::Float64`: New radius used to recluster components of the jet. Defaults to `1.0` if no value is specified.
+
+---
+
+### `JetFilter`
+
+The `JetFilter` structure is used to filter jets based on a specific radius and the number of hardest subjets. This technique reduces contamination from peripheral soft particles.
+
+**Fields**:
+
+- `filter_radius::Float64`: Radius parameter used to recluster subjets.
+- `num_hardest_jets::Int`: Number of hardest subjets retained in the filtered result.
+
+---
+
+### `JetTrim`
+
+`JetTrim` instance is used to trim jets by removing soft, large-angle components from the jet. This is useful in cleaning up jets to remove softer particles at wide angles.
+
+**Fields**:
+
+- `trim_radius::Float64`: Radius used for reclustering in trimming.
+- `trim_fraction::Float64`: Minimum momentum fraction for retained subjets.
+- `recluster_method::JetAlgorithm.Algorithm`: Method identifier for reclustering.
+
+---
+
+## Functions
```

Comment:
Same comment as above - don't repeat docstrings, show how to use these functions.